content/02.introduction.md

```diff
@@ -2,5 +2,94 @@
+Incorporating this edge into Hetionet would be an interesting idea, but it is tangential to this work.
+
+#### Compound Binds Gene
+
+Majority of effort for Compound-binds-Gene extraction focuses on the Biocreative task VI track 5 shared task [@raw:biocreative/chemprot].
+This shared task challenged contestants to classify compound-protein (genes for our context) interactions.
+Curators manually annotated 2,432 PubMed abstracts for five different compound protein interactions (agonist, antagonist, inhibitor, activator and substrate/product production). 
+For this task performance was reported in terms of F1 score and the best team achieved a score of 64.10% [@raw:biocreative/chemprot].
+This competition provided a publicly available dataset that copious amount of supervised machine learning methods use [@tag:limtox; @tag:lptk; @tag:cbg_ensemble_dl; @tag:biobert; @tag:cbg_ensemble_dl; @tag:cbg_feat_engineering; @tag:cbg_transfer_learning; @tag:cbg_neural_attention; @tag:recursive_nn].
+One study used a semi-supervised approach to extract compound-gene interactions [@tag:semi_supervised_vae]. 
+Using already annotated datasets the authors used a variational auto encoder consisting of a bidirectional long short term memory network for encoding and decoding and a convolutional neural network for classification.
+Their method depends on available annotated datasets, which creates a bottleneck for extracting less studied relationships.
+  
+Concerning unsupervised methods, one study used hand crafted rules for CbG extraction [@tag:pharmpresso].
+Another study used a bi-clustering approach on dependency trees  to identify key sentences for CbG extraction [@tag:global_network].
+We used results from the latter study as label functions for CbG extraction.
+
+#### Gene-Gene Interactions
+
+Like Disease-associates-Gene, a plethora of work the for Gene-interacts-Gene (GiG), aka protein-protein interactions, relationship  consisted of co-occurrence approaches.
+One study used a Z-score approach to find significant protein pairs from Pubmed abstracts [@tag:string].
+A later approach builds off of this mentioned method by using distant supervision and a logistic model to improve performance [@tag:cocoscore].
+Following the above methods other, studies used co-occurrence approaches for CbG Extraction [@tag:ppinterfinder; @tag:hpiminer; @protein_protein_co_network; @tag:full_text_co_abstracts].
+Similar to the above relationships, one study used a bi-clustering approach on dependency trees for GiG extraction. [tag:global_network].
+We use the results of this study for our domain heuristic label functions for GiG extraction.
+These methods benefit from not needing annotated data.
+Plus, these methods provide great recall performance in terms of extraction.
+
+Majority of supervised machine learning methods used publicly available datasets [@tag:aimed; @tag:bioinfer; @raw:LLL; @tag:hprd50; @raw:IEPA] as a gold standard.  
```

Comment:
The grammar of this sentence needs a fix - when you do that, can you also note if there are any particular datasets that are most commonly used? Is it just that each team uses a different one, or is there one or two that dominates?